microgen/shape/strut_lattice/cuboctahedron.py

```diff
@@ -0,0 +1,47 @@
+from .abstract_lattice import AbstractLattice
+from itertools import product
+import numpy as np
+import numpy.typing as npt
+import math as m
+from scipy.spatial import KDTree
+
+
+class Cuboctahedron(AbstractLattice):
+    """
+    Class to create a unit cuboctahedron lattice of given cell size and density or strut radius
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs, strut_heights=m.sqrt(2.0) / 2.0)
```

Comment:
I would use math functions from `numpy` since we already import it, thus we would not need to import `math`.
```suggestion
        super().__init__(*args, **kwargs, strut_heights=np.sqrt(2.0) / 2.0)
```

Same suggestion in all the other files.

Reply:
Done. Thanks for notifying me, I'm so used to the math package...